schemas/iso19139/src/main/plugin/iso19139/index-fields/index.xsl

```diff
@@ -1105,6 +1107,8 @@
             <atomfeed><xsl:value-of select="gmd:linkage/gmd:URL"/></atomfeed>
           </xsl:if>
           <link type="object">{
+            "hash": "<xsl:value-of select="digestUtils:md5Hex(string(exslt:node-set(normalize-space(.))))"/>",
```

Comment:
Working on a new indexing module, I think we could simplify this to `digestUtils:md5Hex(normalize-space(.))` which should produce the same hash or is there a particular case where we need exslt and the string function?

Reply:
@fxprunayre you are correct that this seems cleaner.